sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkTracerManagement.java

```diff
@@ -70,4 +71,22 @@
    * @see SpanProcessor#forceFlush()
    */
   CompletableResultCode forceFlush();
+
+  /**
+   * Attempts to stop all the activity for this {@link Tracer}. Calls {@link
+   * SpanProcessor#shutdown()} for all registered {@link SpanProcessor}s.
+   *
+   * <p>This operation may block until all the Spans are processed. Must be called before turning
+   * off the main application to ensure all data are processed and exported.
+   *
+   * <p>After this is called, newly created {@code Span}s will be no-ops.
+   *
+   * <p>After this is called, further attempts at re-using or reconfiguring this instance will
+   * result in undefined behavior. It should be considered a terminal operation for the SDK
+   * implementation.
+   */
+  @Override
+  default void close() {
+    shutdown();
```

Comment:
can you create an issue to make the `shutdown()` return `CompletableResultCode` for consistency? Thanks!

Reply:
Note: #2422 implements this, so I don't know that we need an issue.